### ISSUE TITLE
Introduction: clean up list of available boards

### ISSUE THREAD

**Issue by okin** [40-percent/40percent-wiki#24]
I'm seeing some problems with the boards listed as entries under guides/introduction.

- Some links are 404 (mostly boards vs. Boards in the URL, relative links would be nice here)
- Minivan: link to proper board page
- Phoenix45: p3d is gone and it is not an in-stock
- TW40 aka. Training Wheel 40: no link, probably hard to get now. Maybe Bully could be a good replacement.
- UT47.2 sold out for ages and no restock in sight. I probably would just delete this entry.

**Comment by FrancisUsher** [comment]
Which page is that? Or are you talking about the individual board pages?

**Comment by okin** [comment]
> Which page is that? Or are you talking about the individual board pages?

I'm talking about the page at `Guides/Introduction.md`. There is a board listing at the end.

**Comment by FrancisUsher** [comment]
Oh yeah, that seems like a difficult thing to keep up to date, relevant, and unbiased. I wonder if we could randomly generate some entries at the bottom on each page load

**Comment by okin** [comment]
For starts I'd say lets manually curate this.

Maybe we can use page tags in some way to have boards displayed there. (not sure if jswiki can do this)

**Comment by FrancisUsher** [comment]
Yeah I'm just thinking of strategies for picking boards that aren't going to end up with dead links or unavailable. It's not really an easy problem. Might be something we just want to break out into a separate page.
